### DISCUSSION TITLE
Freeing up RAM

### DISCUSSION THREAD

**Discussion by vinnymoy** [lllyasviel/Fooocus#3505] [Q&A] [answered]
Greetings, I have a question regarding memory management...

I have an AMD Radeon 6700 XT graphics card with 12 GB of dedicated RAM, plus 32 GB of general RAM. I am running the program in local mode, and after generating images, I noticed in the Task Manager that both the graphics card RAM and the general RAM remain full and are not being released... Can the memory be freed without closing the program?

**Comment by mashb1t** [comment, marked as answer]
You may apply https://github.com/lllyasviel/Fooocus/pull/3263/files or https://github.com/lllyasviel/Fooocus/pull/3262/files manually to your code.
